authenticate/handlers.go

```diff
@@ -489,32 +494,34 @@ func (a *Authenticate) isAdmin(user string) bool {
 
 // Dashboard renders the /.pomerium/ user dashboard.
 func (a *Authenticate) Dashboard(w http.ResponseWriter, r *http.Request) error {
+	state := a.state.Load()
+
 	s, err := a.getSessionFromCtx(r.Context())
 	if err != nil {
 		s.ID = uuid.New().String()
 	}
 
-	pbSession, err := session.Get(r.Context(), a.dataBrokerClient, s.ID)
+	pbSession, err := session.Get(r.Context(), state.dataBrokerClient, s.ID)
```

Comment:
I don't think session.Get/Set/Delete are currently handling if `dataBrokerClient` is nil?